### PR TITLE
Implement EditOrderCommand

### LINKED ISSUE #84: Edit order

### PR DESCRIPTION
**Pull request by shavonneg**
Add EditOrderCommand.java by utilising remove and
add functions.

Fixes #84

### CODE REVIEW

File: src/main/java/seedu/address/model/person/UniquePersonList.java [hunk 1]
```diff
@@ -77,24 +77,41 @@ public void setPerson(Person target, Person editedPerson) {
...
-     * @param order        order to be added.
+     * @param order        order to be removed.
      */
-    public void setPersonAndAddOrder(Person target, Person editedPerson, Order order) {
+    public void setPersonAndDeleteOrder(Person target, Person editedPerson, Order order) {
         setPerson(target, editedPerson);
-        internalOrderList.add(order);
+        internalOrderList.remove(order);
+    }
+
```
Comment (rertyy) [inline src/main/java/seedu/address/model/person/UniquePersonList.java:98]: Did you swap the order of the commands or is this just GitHub being funny hahaha

**Comment by rertyy** [comment]
Will need to reverse merge either this or #130 for conflicts

**Comment by rertyy** [comment]
Fixes #84

File: src/main/java/seedu/address/model/person/UniquePersonList.java (hunk 1 above)
Comment (shavonneg) [reply src/main/java/seedu/address/model/person/UniquePersonList.java:98]: i think github just changed it, but shldnt matter too much right HAHAH

File: src/main/java/seedu/address/model/person/UniquePersonList.java [hunk 2]
```diff
@@ -77,24 +77,41 @@ public void setPerson(Person target, Person editedPerson) {
...
+     * @param orderToAdd    order to be added.
+     */
+
+    public void setPersonAndEditOrder(Person person, Person editedPerson, Order orderToDelete, Order orderToAdd) {
+        requireAllNonNull(person, orderToDelete, orderToAdd);
+        setPerson(person, editedPerson);
+        int index = internalOrderList.indexOf(orderToDelete);
+        internalOrderList.set(index, orderToAdd);
+        internalOrderList.remove(orderToDelete);
+        internalOrderList.add(orderToAdd);
```
Comment (rertyy) [inline src/main/java/seedu/address/model/person/UniquePersonList.java:114]: Don't need these 2 lines cos `orderToDelete` will get gc'ed automatically and `orderToAdd` replaces the previous index; we don't want duplicates.

**Review by rertyy — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/person/UniquePersonList.java (hunk 2 above)
Comment (shavonneg) [reply src/main/java/seedu/address/model/person/UniquePersonList.java]: Yep, just saw the error too, fixed!

**Review by rertyy — APPROVED** [review]
LGTM

**Pull request merged**